.appveyor.yml

```diff
@@ -41,7 +41,7 @@ test_script:
       "Running npm run test:without-system-tests..."
       "--------------------------------------------"
       npm run test:without-system-tests
-      #npm run coverage:system-tests
+      npm run coverage:system-tests
```

Comment:
I've confirmed that the system tests are running correctly both on appveyor and locally. I believe the issues seen a few days ago were related exclusively to appveyor since 1) I've confirmed vscode didn't make any changed to the DOM & 2) all system tests leverage the same piece of code that looks for `.monaco-workbench` in the DOM and only the first 2 categories of system tests failed consistently

Reply:
Thanks! Sorry for the confusion. I only saw the `.monaco-workbench` and concluded wrongly that it was a change in the DOM.